LogisticApp/ViewModel/Forms/FormManager.cs

```diff
@@ -57,7 +57,10 @@ public void addMediator(WindowMediator mediator)
 
         private bool canSave(object param)
         {
-            return true;
+            if (_selectedViewModel != null)
+                return _selectedViewModel.canSave();
+            else
+                return true;
```

Comment:
should we return false here?

Reply:
> Those canSave methods can be simplified as shown on the example.
> About those errors they were fixed yesterday but if yhou had not merge main to this branch (I can see you didn't 😺 )
> those fixes won't appear here. After merging this to main branch it all should work!

That's strange, this time i surely was pulling main before applying changes, i created the whole new branch as well, the previous one was named "can_save_method".
Sorry for such a misconception :x

Reply:
We will discuss this later, when I'm back home